Source/ACE.Server/Physics/PhysicsObj.cs

```diff
@@ -1481,10 +1480,10 @@ public void UpdateObjectInternal(double quantum)
                     if (State.HasFlag(PhysicsState.AlignPath))
                     {
                         var diff = newPos.Frame.Origin - Position.Frame.Origin;
-                        newPos.Frame.set_vector_heading(diff.Normalize());
```

Comment:
As already mentioned, change the implementation if needed, not the API

.Normalize() was added for a cleaner API

The implementation is also the same as Vector3.Normalize, the only difference is one uses inlining

If the compiler isn't already inlining these 1-line functions, then the attribute should be added for many of them